packages/opentelemetry-instrumentation-openai-agents/opentelemetry/instrumentation/openai_agents/utils.py

```
@@ -17,3 +19,24 @@ def should_send_prompts():
     env_setting = os.getenv("TRACELOOP_TRACE_CONTENT", "true")
     override = context_api.get_value("override_enable_content_tracing")
     return _is_truthy(env_setting) or bool(override)
+
+
+class JSONEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, dict):
+            if "callbacks" in o:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)